src/main/resources/configCivcraft.yml

```diff
@@ -9487,7 +9472,7 @@ recipes:
     input:
       emeralds:
         material: EMERALD
-        amount: 5
+        amount: 10
```

Comment:
Similar question, why are these more expensive now? Thought we were just reducing..?